randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -319,7 +323,8 @@ Extra - asset_id: 8
   > Door to Forgotten Highway Kago Room
       All of the following:
           Morph Ball
-          Hi-Jump Boots or Space Jump
+          # With Wall Jumps: https://youtu.be/Z_vt91bLKeA
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
```

Comment:
isnt this also possible with gravity+shinespark?

Reply:
Maybe, I'll have to check to see if the ground is level enough to charge a spark.

Reply:
Its a 2 tap shine spark. I feel like anyone who could do this surely can wall jump. Added video though. 
https://youtu.be/a8-2ZAwcMos

Reply:
I'm poretty sure your stutter is at the wrong room lmao
This comment is for Crab maze: door to forgotten highway elbow to door to forgotten highway kago room

Reply:
This is the door to the Kago Room, yeah?

Reply:
Oh my, you're right. Well, this needed to be added anyway.

Reply:
Added shine spark to crab maze